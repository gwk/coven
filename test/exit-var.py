def f(e):
  exit(e)

f(0)
