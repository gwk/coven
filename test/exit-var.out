----------------
Coverage Report:

__main__: exit-var.py:
   1   def f(e):
   2     exit(e)
   3
   4 % f(0)

__main__: exit-var.py: 3 traceable; 2 covered; 0 ignored; 0 ignored but covered; 1 not covered.
